### ISSUE TITLE
🧐[问题 | question] How can we add custom theme Icon in SettingDrawer?

### ISSUE THREAD

**Issue by beeena** [ant-design/ant-design-pro#11341]
提问前先看看：

https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md

**🔩 Module or component**
**In SettingDrawer**

**🥰 Requirement Description**

I want to add a custom theme icon in the SettingDrawer to change the theme, like this.
![image](https://github.com/user-attachments/assets/23cba646-7115-47b6-8d4a-cd55fec7924b)n 

In the picture above, there are three themes. I am trying to add a third custom theme in addition to the light and dark themes. How can I achieve this? I didn't find any such type of SettingDrawer props.

I'm using these versions

```json
"@ant-design/pro-layout": "7.19.11",
"antd": "^5.14.0",
```

**Comment by lihong1232** [comment]
I suppose that Ant doesn't support this function .

**Comment by afc163** [comment]
YES, you have to fork https://github.com/ant-design/pro-components
